### ISSUE TITLE
[Bug] @transifex/cli doesn't find any files on Windows

### ISSUE THREAD

**Issue by mgrilec** [transifex/transifex-javascript#139]
**Current Behavior**
```
$ find . -name *.html | wc -l
1748
$ npx txjs-cli push --token=<redacted> --secret=<redacted>
Parsing all files to detect translatable content...
⸨████████████████████████████████████████⸩ 0/0 {file}
✓ Processed 0 file(s) and found 0 translatable phrases.
✓ Content detected in 0 file(s).
⚠ Nothing to upload.
```

**Expected Behavior**
I expect it to find all the default extension files (html, ts, etc), parse them and upload the translations to our native project.

**Steps to Reproduce**

Run on a windows machine with node > 12.

```
npm install -g @transifex/cli
touch sample.html
txjs-cli push --token=<PROJECT_TOKEN> --secret=<PROJECT_SECRET>
```

I expect this to at least find the html file and process it.

**Possible Solution**
Passing in the whole absolute path with forward slashes to a single file works.

**Possible Implementation**
Looks like there's an issue with how the absolute path is returned on windows and how the glob package expects paths.

https://github.com/transifex/transifex-javascript/blob/f28ea0561fdefa101ca5801a41d96f91fff44b93/packages/cli/src/commands/push.js#L35

shelljs.pwd returns the absolute paths with backslashes on Windows, while the glob package works only with forward slashes.

https://github.com/isaacs/node-glob/blob/af57da21c7722bb6edb687ccd4ad3b99d3e7a333/README.md?plain=1#L324

**Comment by nbasili** [comment]
We have released [v4.2.1](https://github.com/transifex/transifex-javascript/releases/tag/v4.2.1) which should fix the path issue on Windows.

**Comment by mgrilec** [comment]
Works for me.
